lib/cutting_edge/workers/badge.rb

```diff
@@ -21,8 +21,7 @@ class BadgeWorker < GenericWorker
   def perform(identifier)
     log_info 'Running Worker!'
     dependencies = get_from_store(identifier)
-    
-    if dependencies && !dependencies.empty?
+    if dependencies && !dependencies.empty? && !(dependencies[:outdated] == :unknown)
```

Comment:
What prompted this additional check for `:unknown`?

Reply:
When trying to access a private repo, `dependencies` will exist in the store, and not be `empty?` because it contains stats (namely, that it's status is `:unknown`). So no error badge was being thrown (just a badge with the 'CE' part, but no numbers).